src/layout/Number/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { forwardRef } from 'react';
+
+import { NumberDef } from 'src/layout/Number/config.def.generated';
+import { NumberComponent } from 'src/layout/Number/NumberComponent';
+import type { PropsFromGenericComponent } from 'src/layout';
+
+export class Number extends NumberDef {
+  render = forwardRef<HTMLElement, PropsFromGenericComponent<'Number'>>(
+    function LayoutComponentNumberRender(props, _): JSX.Element | null {
+      return <NumberComponent {...props} />;
+    },
+  );
+}
```

Comment:
This should probably also implement the `DisplayData<'Number'>` interface, so that the numeric display data can be referenced in an expression. :raised_hands: